xtevent/_eventiv.ado

```diff
@@ -169,7 +179,7 @@ program define _eventiv, rclass
 	loc komit: list uniq komit		
 	
 	if "`gen'" != "nogen" {	
-		_eventgenvars if `touse', panelvar(`panelvar') timevar(`timevar') policyvar(`policyvar') lwindow(`lwindow') rwindow(`rwindow') `trend' norm(`norm') `staggered'
+		_eventgenvars if `touse', panelvar(`panelvar') timevar(`timevar') policyvar(`policyvar') lwindow(`lwindow') rwindow(`rwindow') `trend' norm(`norm') impute(`impute')

```

Comment:
@Constantino-Carreto-Romero if the user chose a imputation scheme, then `_eventgenvars` was already executed above, and it is inefficient to call it again, because you may already have the event dummies you have in the dataset.

Reply:
In the case of _eventiv.ado, I had to do the call of _eventgenvars.ado in two parts. First, I call the part that only does the imputation, and the imputed policyvar is used in _eventiv.ado for finding the leads of proxyiv and this in turn redefines the coefficients to normalize. Then, I call only the second part of _eventgenvars.ado, which generates the event-time dummies. This second part uses as one of the inputs the coeffcients that will be normalized.